### PR TITLE
JSON encode/decode Node() in set / get

### PR DESCRIPTION
**Pull request by Casperhr**
- CacheSessions is now working
 - Option to cache all Node() instead of only Node(String)

### CODE REVIEW

File: Sources/VaporRedis/RedisCache.swift [hunk 1]
```diff
@@ -1,74 +1,72 @@
...
+   	public func get(_ key: String) throws -> Node? {
         guard let result = try redbird.command("GET", params: [key]).toMaybeString() else {
             return nil
         }
-
-        return Node.string(result)
-	}
-
+            
+        return try JSON(bytes: result.bytes).makeNode()["data"]?.node
```
Comment (loganwright) [inline Sources/VaporRedis/RedisCache.swift]: do we need `?.node` here? I believe it should return an optional node from `["data"]`

File: Sources/VaporRedis/RedisCache.swift [hunk 2]
```diff
@@ -1,74 +1,72 @@
...
-
-        try redbird.command("SET", params: [key, string])
-	}
-
+     Sets a key to the supplied value in the
+     underlying Redbird instance using the
+     SET command.
+     */
+    public func set(_ key: String, _ value: Node) throws {
+        try redbird.command("SET", params: [key, JSON(node: Node(["data": value])).serialize().toString()])
```
Comment (loganwright) [inline Sources/VaporRedis/RedisCache.swift]: should be able to use `node: ["data": value]` directly w/ the external param there, let me know if you're not seeing that.

**Review by loganwright — APPROVED** [review]
I think the comments styling in here doesn't match the format of the other ones in Vapor for indentation.

Otherwise, looks good to me. @tannernelson thoughts?

File: Sources/VaporRedis/RedisCache.swift (hunk 1 above)
Comment (Casperhr) [reply Sources/VaporRedis/RedisCache.swift]: fixing this

File: Sources/VaporRedis/RedisCache.swift (hunk 2 above)
Comment (Casperhr) [reply Sources/VaporRedis/RedisCache.swift]: Not sure I understand this?

**Pull request merged**